community/release.md

```diff
@@ -30,20 +49,25 @@ The release of Capact container images and binaries is automated and done using
 1. Click `Run workflow` and depending on your scenario:
    - if making a major/minor release from `main`, select the `main` branch and put the version in SemVer, e.g. `0.5.0`,
    - if making a patch release from a release branch, select the given release branch and put the version in SemVer, e.g. `0.5.1`.
+   - provide Dashboard image tag from a [`capactio/dashboard`](https://github.com/capactio/dashboard) release build.
 1. Click `Run workflow` to start the release workflow.
 
 The workflow will prepare the release branch, tag the appropriate commit and create a GitHub Release. [`gren`](https://github.com/github-tools/github-release-notes) is used to create the release notes from merged pull requests.
 
-> *NOTE:* After the release is complete consider removing the `GH_PAT` from the Release environment.
-
-### Prepare Hub for release 
+### Release Hub manifests 
 
 The release of Hub Manifests is automated and done using a GitHub Action workflow.
 
 1. Open the [Make release](https://github.com/capactio/hub-manifests/actions/workflows/make-release.yaml) workflow.
-1. Click `Run workflow` and put the version parameter and the runner image tag from a [`capactio/capact`](https://github.com/capactio/capact) build.
+1. Click `Run workflow` and put the version parameter and the runner image tag from a [`capactio/capact`](https://github.com/capactio/capact) release build.
 1. Click `Run workflow` to start the release workflow.
 
+### Validate the release
+
+1. Install Capact locally from the version you've just released.
+1. Run one of our manifests, such as Mattermost installation. Make sure it passes.
+1. Edit the draft release with key highlights and publish it on the [`capact` Releases](https://github.com/capactio/capact/releases) page.
```

Comment:
Consider moving 
> publish it on the [`capact` Releases](https://github.com/capactio/capact/releases) page.

to the **Further manual steps** section right before 3rd step.